src/plugins/liveobjects/batchcontextlivecounter.ts

```
@@ -0,0 +1,38 @@
+    this._batchContext.queueStateMessage(stateMessage);
+  }
+
+  decrement(amount: number): void {
+    this._batchContext.throwIfClosed();
+    // do an explicit type safety check here before negating the amount value,
+    // so we don't unintentionally change the type sent by a user
+    if (typeof amount !== 'number') {
+      throw new this._client.ErrorInfo('Counter value decrement should be a number', 40013, 400);
+    }
+
+    return this.increment(-amount);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)